schema/home/fetch.js

```diff
@@ -21,14 +29,28 @@ export const featuredAuction = () => {
   });
 };
 
+export const followedGenes = (accessToken, size) => {
+  return gravity.with(accessToken)('me/follow/genes', { size });
+};
+
 export const featuredGene = (accessToken) => {
-  return gravity.with(accessToken)('me/follow/genes', { size: 1 }).then((follows) => {
+  return followedGenes(accessToken, 1).then((follows) => {
     if (follows.length) {
       return first(follows).gene;
     }
   });
 };
 
+export const geneArtworks = (id, size) => {
+  return gravity('filter/artworks', {
+    gene_id: id,
+    for_sale: true,
+    size: 60,
```

Comment:
Yeah this should really be shuffled on Gravity and then just return the 20 or so we really want.